src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -105,13 +98,9 @@ class ContentSidebar extends PureComponent<Props, State> {
         getPreviewer: noop,
         currentUser: undefined,
         hasSkills: false,
-        hasProperties: false,
         hasMetadata: false,
-        hasNotices: false,
```

Comment:
Will need to figure out the best place to default these. Probably in DetailsSidebar

Reply:
default these here itself since they are needed in this file